modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js

```diff
@@ -44,6 +45,12 @@ class RuleList extends Component {
 				true
 			)
 		);
+
+		var tooltip = ClayTooltip.init();
+
+		setTimeout(() => {
+			tooltip.selectors = ['.invalid-rule'];
+		}, 1500);
```

Comment:
@alinedoleron, can you please clarify why this is necessary?

Reply:
I thought it was necessary for the tooltip appears smoothly, like it is done on [Clay's demo](https://github.com/liferay/clay/blob/2.x/packages/clay-tooltip/demos/index.html). But I removed to test now and it behaves the same way as before. So it's not necessary indeed. Thank you.